### DISCUSSION TITLE
Starting the language service process timed out

### DISCUSSION THREAD

**Discussion by 865881900** [eslint/eslint#15352] [Q&A] [answered]
mac 10.15.7
node v16.13.0/v10.19.0/v14.17.0
eslint ^7.32.0
webstrom v4.1.7
error: ESlint :Starting the language service process timed out

This error also occurred after I downloaded the value of this project. I tried to switch the version of Node. But it didn't solve the problem;
I don't even know what to do with it

**Comment by 865881900** [comment]
I just installed the latest version of Node on the node website, thinking it was NVM.
There are no nodes managed by NVM

**Comment by nzakas** [comment, marked as answer]
ESLint doesn’t contain a language server. This error is likely coming from VS Code.

**Reply by 865881900** [reply]
I will try to upgrade VS Code to try, thank you

**Reply by 865881900** [reply]
After reinstalling Vs Code, I fixed the problem.
